doc/source/management/data_at_rest_encryption.rst

```diff
@@ -220,18 +192,47 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. _ps.data-at-rest-encryption.redo-log:
+
+Redo Log Encryption
+================================================================================
+
+:Availability: This feature is **Alpha** quality
+
+InnoDB redo log encryption is enabled by setting the variable
+``innodb_redo_log_encrypt``. This variable has two values: ``MASTER_KEY`` and
```

Comment:
Since the feature is not mandatory, there must be an "off" value as well

Reply:
Default value added